apps/reference/docs/guides/database.mdx

```diff
@@ -122,120 +122,7 @@ you can update the replica identity of your tables, setting it to `full`:
-
-const OLD_PROJECT_URL = 'https://xxx.supabase.co'
-const OLD_PROJECT_SERVICE_KEY = 'old-project-service-key-xxx'
-
-const NEW_PROJECT_URL = 'https://yyy.supabase.co'
-const NEW_PROJECT_SERVICE_KEY = 'new-project-service-key-yyy'
-
-;(async () => {
-  const oldSupabaseRestClient = createClient(
-    OLD_PROJECT_URL,
-    OLD_PROJECT_SERVICE_KEY,
-    {
-      schema: 'storage',
-    }
-  )
-  const oldSupabaseClient = createClient(
-    OLD_PROJECT_URL,
-    OLD_PROJECT_SERVICE_KEY
-  )
-  const newSupabaseClient = createClient(
-    NEW_PROJECT_URL,
-    NEW_PROJECT_SERVICE_KEY
-  )
-
-  // make sure you update max_rows in postgrest settings if you have a lot of objects
-  // or paginate here
-  const { data: oldObjects, error } = await oldSupabaseRestClient
-    .from('objects')
-    .select()
-  if (error) {
-    console.log('error getting objects from old bucket')
-    throw error
-  }
-
-  for (const objectData of oldObjects) {
-    console.log(`moving ${objectData.id}`)
-    try {
-      const { data, error: downloadObjectError } =
-        await oldSupabaseClient.storage
-          .from(objectData.bucket_id)
-          .download(objectData.name)
-      if (downloadObjectError) {
-        throw downloadObjectError
-      }
-
-      const { _, error: uploadObjectError } = await newSupabaseClient.storage
-        .from(objectData.bucket_id)
-        .upload(objectData.name, data, {
-          upsert: true,
-          contentType: objectData.metadata.mimetype,
-          cacheControl: objectData.metadata.cacheControl,
-        })
-      if (uploadObjectError) {
-        throw uploadObjectError
-      }
-    } catch (err) {
-      console.log('error moving ', objectData)
-      console.log(err)
-    }
-  }
-})()
-```
-
-#### Caveats
-
-- The new project will have the old project's Storage buckets, but not the objects. You will need to migrate Storage objects manually.
-
-### Resetting your project password
-
-When you create a new project in Supabase we ask for a password. You can use this password to connect directly to your Postgres database.
-
-If you forget your password, you can reset it from the Dashboard under the database settings page.
-
-Read more in [Database Configuration](/docs/guides/database/managing-passwords).
-
-### Changing the timezone of your server.
-
-Your database is initialized with the UTC timezone. We recommend keeping it this way, as it is helpful for time calculations.
-If, however, you want to update the timezone, you can do so using any of the [database timezones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
-
-For example:
-
-```sql
-alter database postgres set timezone to 'America/New_York';
-```
-
-Read more in [Database Configuration](/docs/guides/database/managing-timezones).
+Read about resetting your database password [here](/docs/guides/database/managing-passwords) and changing the timezone of your server [here](/docs/guides/database/managing-timezones).
```

Comment:
Did you mean to put this in the "migrating between projects" doc? Seems a bit out of place here:
![Screen Shot 2022-09-21 at 11 57 37 AM](https://user-images.githubusercontent.com/7026076/191587870-088fe12e-2342-4463-b4e3-6c18bbd7ca66.png)

Reply:
This will be merged with the realtime guide [here](https://supabase.com/docs/guides/realtime/postgres-changes). This is on @w3b6x9's radar already